scripts.js

```diff
@@ -1,34 +1,45 @@
 $('main').append(
-    '<h3 id="new">Meglepiiii</h3>'
```

Comment:
Erre gondoltam hogy be lehet állítani. Ezt hívják indent-nek. VS Code/Settings itt pedig írd be az indent keresőszót, Editor: tab size legyen 4, mint a képen
![image](https://user-images.githubusercontent.com/46938873/79066537-ea99dc00-7cb8-11ea-8846-d18c39d50831.png)